cfgov/unprocessed/apps/analytics-gtm/js/accessibility-interactions-listeners.js

```diff
@@ -5,8 +5,8 @@ import {
 
 const AccessibleInteractionsListeners = ( () => {
 
-  // Listen for focus on the "skip link" button.
-  const skipLinkButton = document.querySelector( '#skip-nav' );
+  // Listen for focus on the "skip link" button. This will grab the top link.
```

Comment:
Do we not want this to be able to handle additional skip links, too?

Reply:
The purpose of this script is to see whether someone tabbed into the page from the start, so it should be fine triggering on just the first skip link. If we want both we need to set up an tag for it in google tag manager. I'll check with analytics, but it wouldn't be needed for this PR and could be its own thing, if analytics wants it.

Reply:
Cool. Might be worth tweaking the comment slightly to make that more clear.

Reply:
Done